Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.7"
 
 [deps]
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
```

Comment:
This is quite heavy, we should make it into an extension

Reply:
I don't think we even need that, I was using it for `Uniform` and now it's only needed for `Bernoulli`, so I'm sure we can avoid it all together. I just needed to get the tests running to catch the various CUDA errors